src/manage_network_info/conf.js

```diff
@@ -0,0 +1,12 @@
+const path = require("path")
```

Comment:
I'd be find having this file on the top level, since it's a confid somewhat similar to truffle-config.js. Would also save you a bunch of ../../